### ISSUE TITLE
IOS- CSS transforms ignored during screenshot

### ISSUE THREAD

**Issue by DhiaNHM** [Telerik-Verified-Plugins/NativePageTransitions#167]
Hi,

We have a Cordova 6.5 app that uses Ionic and swiperJS, On IOS the inline transforms applied by SwiperJS are reset at the point the transition plugin takes it's screenshot so the stacked 3D effect of the swiper image s is lost during transition, Android however behaves correctly and the transforms are retained.

Is this a known IOS webview issue? any workarounds?

Thanks,

**Comment by videmort** [comment]
Plugin is not updated so... is not working on Cordova 6.0 + and iOS engine 4.1 +

**Comment by fishme** [comment]
that is not right - I'm working with Cordova 7.1 and iOS 4.4.0 and everything is working perfect.
